recipes-devtools/firmware-package-tool/files/update.conf.json

```diff
@@ -0,0 +1,34 @@
+{
+  "firmware": [
+    {
+      "description": "IOT2050 PG1 Bootloader Release V01.01.01",
+      "name": "iot2050-pg1-image-boot.bin",
+      "version": "V01.01.01",
+      "target_boards": [
+        "SIMATIC IOT2050-BASIC",
+        "SIMATIC IOT2050 Basic",
+        "SIMATIC IOT2050-ADVANCED",
+        "SIMATIC IOT2050 Advanced"
+      ]
+    },
+    {
+      "description": "IOT2050 PG2 Bootloader Release V01.01.01",
+      "name": "iot2050-pg2-image-boot.bin",
+      "version": "V01.01.01",
```

Comment:
Version must be generated, likely derived from `git describe` (see `ISAR_RELEASE_CMD`).

Reply:
This is raw json, which provides formatting. The final version value will be replaced by ISAR_RELEASE_CMD. Please check [iot2050-fwu-update-json.sh ](https://github.com/siemens/meta-iot2050/pull/446/commits/95c6a370ae3e0e65bba2bfc8985f334230e19696#diff-0c9ec4ecf7b23e3159a1bac613fc1112d4338a917ec3698a6d00d71944214b50).